operator/images/cluster-setup/content/bin/install.sh

```diff
@@ -127,6 +127,9 @@ install_clusters() {
     printf -- "- Installing applications via Openshift GitOps... \n"
     install_applications | indent 4
 
+    printf -- "- Checking application status\n"
+    check_applications "openshift-gitops" "pipeline-service" | indent 4
```

Comment:
Also since we have check_applications now, do we still need check_deployments ?
My understanding is that argo apps are "synced" and "healthy" only if all the deployments are ready right ?
If that's the case then we can get rid of check_deployments.

Reply:
I think we would need to verify that the "AtLatestKnown" status is all we need to verify health. That may not be the case - for instance, OLM can say it installed the operator, but the operator can report that a deployment it manages ("operand") is not healthy.